__tests__/magic-block-parser.test.js

```diff
@@ -240,9 +240,10 @@ describe('Parse Magic Blocks', () => {
   it('Custom HTML Block', () => {
     const text = `[block:html]
     ${JSON.stringify({
-      html: '<h1>👋🌍</h1>\n<hr>\n<form>\n  <input name="test" value="hello" type="text"/>\n  <br/>\n  <a class="button">Go</a>\n</form>',
```

Comment:
Our users probably have inputs and forms in these custom HTML block which they'll expect to work… 😬